client/src/pages/Doctorpage.js

```diff
@@ -1,12 +1,98 @@
-import React from 'react';
```

Comment:
can you rename `Doctorpage.js` to `DoctorPage.jsx`. #Pascal case

Reply:
its done.